### PR TITLE
Bug/1931 dont export invisible fields

### PR DESCRIPTION
**Pull request by steventlamb**

### CODE REVIEW

File: opentreemap/exporter/tasks.py [hunk 1]
```diff
@@ -31,14 +33,18 @@ def extra_select_and_values_for_model(
...
+    for perm in (perm for perm in perms if perm.permission_level > 0):
         field_name = perm.field_name
         prefixed_name = prefix + field_name
 
         if field_name.startswith('udf:'):
             name = field_name[4:]
             extra_select[prefixed_name] = "%s.udfs->'%s'" % (table, name)
+        else:
+            if not model_hasattr(dummy_instance, field_name):
+                continue
```
Comment (maurizi) [inline opentreemap/exporter/tasks.py]: Consider adding a comment here as to why this is being done.

File: opentreemap/exporter/tasks.py [hunk 2]
```diff
@@ -31,14 +33,18 @@ def extra_select_and_values_for_model(
 
     extra_select = {}
     prefixed_names = []
+    dummy_instance = safe_get_model_class(model)()
 
-    for perm in perms:
+    for perm in (perm for perm in perms if perm.permission_level > 0):
```
Comment (maurizi) [inline opentreemap/exporter/tasks.py]: Consider changing `if perm.permission_level > 0` to `if perm.permission_level >= FieldPermission.READ_ONLY`

**Comment by maurizi** [comment]
Other than the 2 comments I left, this looks correct to me, though I have not tested it.

File: opentreemap/exporter/tasks.py (hunk 2 above)
Comment (steventlamb) [inline opentreemap/exporter/tasks.py]: great idea

File: opentreemap/exporter/tasks.py (hunk 1 above)
Comment (steventlamb) [inline opentreemap/exporter/tasks.py]: will do

**Comment by maurizi** [comment]
Also, according to the Travis build it looks like you broke a couple of unit tests

**Comment by steventlamb** [comment]
@maurizi this has been overhauled to address your comments, fix the unit tests, and fix some issues I found along the way. I tested the CSV exports for perms, it works in my env. I think it's ready to merge.

**Comment by maurizi** [comment]
Looks good to me (still haven't tested it)

**Pull request merged**